<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SingleDeodorationFan</class>
 <widget class="QWidget" name="SingleDeodorationFan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>111</width>
    <height>80</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="indexLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>20</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>15</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>#</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>40</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>引风机</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fanPushButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>29</y>
     <width>111</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
